docs/testing.md

```diff
@@ -0,0 +1,78 @@
+CRI-Containerd Testing Guide
+============================
+This document assumes you have already setup the development environment (go, git, `cri-contaienrd` repo etc.).  
+ 
+Before sending pull requests you should at least make sure your changes have passed code verification, unit test and CRI validation test.
+## Code verification
+Code verification enforces some styles on go code, boilterplate etc.
+
+* Install tools used by code verification:
+```shell
+make install.tools
+```
+* Run code verification:
+```shell
+make verify
+```
+## Unit test
+Run all unit tests in `cri-containerd` repo.
+```
+make test
+```
+## CRI validation test
+[CRI validation test](https://github.com/kubernetes/community/blob/master/contributors/devel/cri-validation.md) is a test framework validating that a Container Runtime Interface (CRI) implementation meets all the requirements.
```

Comment:
yeah so is CRI Validation Test "make test-cri" for cri-containerd or is it cri-tools's `critest validation` as configured to verify a particular CRI implementation? I presumed you meant `make test-cri` for cri-containerd.